brambling/static/brambling/sass/modules/_list-item.sass

```diff
@@ -0,0 +1,5 @@
+.list-item
+	display: flex
+	flex-direction: row
+	justify-content: space-between
+	align-items: center
```

Comment:
I'm not sure if it's safe for us to use flexbox on the site? @nmorduch says it should be fine for non-essential purposes. Not sure whether this change counts as non-essential?